packages/cells/src/widget.ts

```diff
@@ -414,6 +414,11 @@ export class Cell<T extends ICellModel = ICellModel> extends Widget {
       const layout = this._inputWrapper!.layout as PanelLayout;
       if (value) {
         this._input!.parent = null;
+        if (this._inputPlaceholder) {
+          this._inputPlaceholder.text = this.model.sharedModel
```

Comment:
I would advice to restore the model change signal because in RTC the source may change due to another user.

Reply:
I think I might not understand what you're going for - we can't make edits when the placeholder is shown (in a single-user context) so it would never update the placeholder text (for single users)

Reply:
Yes this is true. My comment is exactly for the case of multiple users editing the same document. The content may be changed by other users.

Reply:
Oh ok so do you think I should update it in the `set outputHidden` function how I am [here](https://github.com/jupyterlab/jupyterlab/pull/14141/files#diff-352916c4cc928b1298618fffc0424a71ee8816925f3a36c77d8593057e7b6f89R1181)?

Reply:
BTW added that back in - let me know how it looks!